bolt/internal/internal.go

```diff
@@ -98,6 +98,14 @@ func MarshalLayout(l chronograf.Layout) ([]byte, error) {
 			}
 		}
 
+		axes := make(map[string]*Axis, len(c.Axes))
+		for a, r := range c.Axes {
```

Comment:
I think something on-target with what's currently there would be better. Such as `b`?  Maybe not the best since it's not just bounds, but super minor.